test/fennec/udp_test.exs

```diff
@@ -228,8 +230,12 @@ defmodule Fennec.UDPTest do
   end
 
   defp udp_communicate(udp, client_id, req) do
+    with_mock Fennec.Auth, [:passthrough], [
```

Comment:
Is there a possibility to not use mocks here? As I see, `Fennec.Auth.maybe` checks only params class and method.